tests/codebuild/testfiles/xgboost-model.yaml

```diff
@@ -5,8 +5,9 @@ metadata:
 spec:
   primaryContainer:
     containerHostname: xgboost
-    modelDataUrl: s3://{DATA_BUCKET}/batch-transform/xgboost-mnist/model.tar.gz
```

Comment:
can you please undo all changes to this file

I have included the `mode: MultiModel` in `tests/codebuild/testfiles/xgboost-hosting-deployment.yaml`